Source/WebKit/WebProcess/WebPage/gtk/AcceleratedSurfaceDMABuf.h

```diff
@@ -132,7 +133,7 @@ class AcceleratedSurfaceDMABuf final : public AcceleratedSurface, public IPC::Me
     class RenderTargetTexture final : public RenderTarget {
     public:
         static std::unique_ptr<RenderTarget> create(uint64_t, const WebCore::IntSize&);
-        RenderTargetTexture(uint64_t, const WebCore::IntSize&, WTF::UnixFileDescriptor&&, unsigned, uint32_t format, uint32_t offset, uint32_t stride, uint64_t modifier);
+        RenderTargetTexture(uint64_t, const WebCore::IntSize&, unsigned, uint32_t format, Vector<WTF::UnixFileDescriptor>&&, Vector<uint32_t>&& offsets, Vector<uint32_t>&& strides, uint64_t modifier);
```

Comment:
Adding the name texture to the unsigned parameter I think it is a good addition for understanding.